src/index.ts

```diff
@@ -92,6 +92,9 @@ export const outlook = (calendarEvent: CalendarEvent): string => {
     location: event.location,
     allday: event.allDay || false,
   };
+  if (event.guests && event.guests.length) {
```

Comment:
I could also extract this to a `formatGuestEmails` and re-use it in all places if you want as we are doing this exact same thing in 6 places